requirements.txt

```diff
@@ -1,3 +1,3 @@
-raiden==1.1.0
-
+-e git+https://github.com/raiden-network/raiden@raiddit#egg=raiden
+-e git+https://github.com/raiden-network/raiden-contracts@raiddit#egg=raiden-contracts
```

Comment:
Does that work for you? I had to use `git+https://github.com/raiden-network/raiden.git@raiddit`

Reply:
it seems (tm) to work here